matlab_simulation/code v1.0/ME597-5-ExtendedKalmanFilter/omnibot_linearize_sensor_model.m

```diff
@@ -0,0 +1,4 @@
+function Ht = omnibot_derivative_sensor_model(mu)
+    n = length(mu);
+    Ht = eye(n);
+end
```

Comment:
There is a mismatch here between the name of the file and the name of the function. The function should just be called omnibot_linearize_sensor_model().